module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupMember.md

```diff
@@ -30,23 +31,28 @@ Add-EntraBetaGroupMember
 ```
 
 ## Description
-The Add-EntraBetaGroupMember cmdlet adds a member to a group.
+
+The `Add-EntraBetaGroupMember` cmdlet adds a member to a group. Specify the `ObjectId` and `RefObjectId` parameters to add a member to a group.
```

Comment:
You could explain what `ObjectId` and `RefObjectId` are similar to what was done [here](https://github.com/microsoftgraph/entra-powershell/blob/main/module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraDeviceRegisteredOwner.md#example-1)

Reply:
File updated.